mozilla/fx-private-relay
# Premium waitlist

# New feature description

This adds a page for people to sign up for the waitlist in countries where Premium is not yet available (MPP-1453). This allows us to both get insight into where demand for Premium is highest, and a way to notify people when Premium becomes available for them.

# Screenshot (if applicable)

You can see it here: https://deploy-preview-1846--fx-relay-demo.netlify.app/premium/waitlist/

# How to test

Navigate to `/premium/waitlist` (e.g. on [the deploy preview](https://deploy-preview-1846--fx-relay-demo.netlify.app/premium/waitlist/)). I don't know how to access the email systems ourselves, so we'll have to verify this again on stage, but I've gotten confirmation that the data shows up [in CTMS](https://mozilla.slack.com/archives/C013CSYEL5T/p1648656495665999?thread_ts=1648225064.770339&cid=C013CSYEL5T) and [in Basket](https://mozilla.slack.com/archives/C013CSYEL5T/p1648645738704949?thread_ts=1648225064.770339&cid=C013CSYEL5T).

# Checklist

- [ ] l10n dependencies have been merged, if any. Submitted here: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/75
